tests/api.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { faker } from '@faker-js/faker/locale/de';
+import { test, expect } from '@playwright/test';
+import { LoginPage } from '../pages/LoginView.page';
+import { LandingPage } from '../pages/LandingView.page';
+import { StartPage } from '../pages/StartView.page';
+
+const PW = process.env.PW;
+const USER = process.env.USER;
+const FRONTEND_URL = process.env.FRONTEND_URL;
```

Comment:
```suggestion
const FRONTEND_URL = process.env.FRONTEND_URL || '';
```
Nicht zwingend notwendig, aber dann meckert der Editor nicht mehr